web/client/components/geostory/media/Video.jsx

```diff
@@ -226,11 +234,22 @@ const VideoMedia = ({
     const isVisible = containerInView && inView;
 
     const [playing, setPlaying] = useState(false);
+    const [mute, setMute] = useState(muted);
 
     const handleOnPlay = (newPlaying) => {
         setPlaying(newPlaying);
         onPlay(newPlaying);
+
     };
+    // if autoplay is true and mute false, the mute status
+    // is forced to true, to handle the error exception in mp4 videos,
+    // in line with the Autoplay Policy, so the video can start
+
+    useEffect(() => {
```

Comment:
I'm still getting the autoplay error.
Steps:
- create a new geostory
- add a video src with mp4
- add video as background for the first cover background
- save the geostory
- open the share link from the share panel

current result
- autoplay + audio error

expected result
- the paused video

The approach proposed in the PR will always mute the autoplay video even if there was interaction in the page. If we decide to follow this path also the UI needs to be updated accordingly.
If possible we should keep the UI as is now and we should try to make it works as usual if there was interaction in the page.

With some test it seems if we introduce an additional internal mute value in the Video component we should be able to achieve the desidered behaviour. We could try something like this:

```js
const Video = withResizeDetector(({
    src,
    width,
    height,
    resolution,
    thumbnail,
    controls,
    play,
    onPlay = () => {},
    onStart = () => {},
    fit,
    loop,
    volume = 1,
    muted
}) => {

    const playing = play;
    // ...
    const [mute, setMute] = useState(false); // internal mute

    // ...

    useEffect(() => {
        if (!started && playing) {
            setStarted(true);
            setLoading(true);
            setMute(true); // set to true if it's playing and it's not started
        }
        if (started) {
            onStart(started);
        }
    }, [started, playing]);
    // ...
    return (
        <div
            className="ms-video"
            //...
        >
            {src &&
            <>
                {started && <ReactPlayer
                    // ...
                    muted={mute || muted} // add the internal mute to the react-player control
                    // ...
                    onPlay={() => {
                        onPlay(true);
                        setMute(false); // disable the internal mute as soon it's start to play and it will pause if there are not interaction 
                    }}
                    // ...
```
There are still issue with corner case(1) but for now it's the only solution I can think of.
(1) background with autoplay and audio